### DISCUSSION TITLE
[@types/aria-query] `ARIARoleDefintionKey` should be `ARIARoleDefinitionKey`

### DISCUSSION THREAD

**Discussion by jsoref** [DefinitelyTyped/DefinitelyTyped#62906] [Issues with a @types package]
https://github.com/DefinitelyTyped/DefinitelyTyped/search?q=ARIARoleDefintionKey

Identified in https://github.com/sveltejs/svelte/pull/7974#discussion_r1005873755

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
I just opened #62916
